test/CodeCoverage/codecoverage.bat

```diff
@@ -1,5 +1,7 @@
 SET WORKINGDIR=%~dp0
 SET REPOROOT=%WORKINGDIR%..\..
+SET SQLTOOLSSERVICE_EXE=%REPOROOT%\src\Microsoft.SqlTools.ServiceLayer\bin\Integration\netcoreapp3.1\win7-x64\MicrosoftSqlToolsServiceLayer.exe
```

Comment:
I would instead just make a var for the netcoreapp3.1 part and then re-use that in all the places that value is used. For now that's fine since everything is compiled against the same version.